python/delta/tables.py

```diff
@@ -25,17 +25,23 @@
 class DeltaTable(object):
     """
         Main class for programmatically interacting with Delta tables.
-        You can create DeltaTable instances using the class methods.
+        You can create DeltaTable instances using the path of the Delta table.::
```

Comment:
What's this ".::" mean?

Reply:
sphinx syntax to say the next lines is a code snippet and therefore format it.